src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -72,18 +161,33 @@ internal class LinuxAppServiceLogger
             // Initialize file logger, if in Linux Dedicated
             if (!writeToConsole)
             {
-                var tenMbInBytes = 10000000;
-                Serilog.Log.Logger = new LoggerConfiguration()
-                    .WriteTo.Async(a =>
-                    {
-                        a.File(
-                            LinuxAppServiceLogger.LoggingPath,
-                            outputTemplate: "{Message}{NewLine}",
-                            fileSizeLimitBytes: tenMbInBytes,
-                            rollOnFileSizeLimit: true,
-                            retainedFileCountLimit: 10);
-                    })
-                    .CreateLogger();
+                // Since it's unreliable to load xml files at runtime in a library, we hardcode the config as a string
+                var xmlStr = "" +
+                    "<?xml version=\"1.0\" encoding=\"utf - 8\" ?>" +
+                    "<log4net> " +
+                    "<appender name=\"RollingFile\" type=\"log4net.Appender.RollingFileAppender\">" +
+                    "<file value=\"" + LoggingPath + "\" />" +
+                    "<layout type=\"log4net.Layout.PatternLayout\">" +
+                    "<conversionPattern value=\"%message%newline\" />" +
+                    "</layout>" +
+                    "<appendToFile value=\"true\" />" +
+                    "<rollingStyle value=\"Size\" />" +
+                    "<maxSizeRollBackups value=\"1\" />" +
+                    "<maximumFileSize value=\"300MB\" />" +
+                    "<staticLogFileName value=\"true\" />" +
+                    "</appender>" +
+                    "<root>" +
+                    "<level value=\"ALL\" />" +
+                    "<appender-ref ref=\"RollingFile\" />" +
+                    "</root>" +
+                    "</log4net>";
+                var xml = new XmlDocument();
+                xml.LoadXml(xmlStr);
+
```

Comment:
I'd like to know if we're ok with maximumFileSize being set to 300MB. If so, after these currently running tests pass, I'm pretty sure we're ready to release

Reply:
@ConnorMcMahon

Reply:
I don't necessarily feel comfortable setting 300MB. I'm checking what LWAWS uses, but the biggest I have seen so far is 10MB.

Reply:
resolving since we're using another logger now